src/styles/theme.js

```diff
@@ -6,5 +6,8 @@ const theme = {
   display: flex;
   justify-content: ${justifyContent};
   align-items: ${alignItem};`,
+  setFlexColumn: (direction = 'column') => `
+  display: flex;
+  flex-direction: ${direction};`,
```

Comment:
column으로만 할거라면, 굳이 함수형태를 가질 필요가 있을까요?column은 기본값으로 두고 justify-content, align-items를 인자로 받는게 맞아보입니다!
또는 이렇게 두개 속성만 믹스인으로 활용할거라면 바로 
```suggestion
  setFlexColumn:`
  display: flex;
  flex-direction: column;
`
```
이렇게 해도 되보이고요!

Reply:
앗 제가 고민하던 부분인데 감사합니다!